Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -42,8 +42,9 @@ public async Task InitDefaultCustomCommands()
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Messaging.BotCommands.SendCommand", @"-send\s*(?<Channel>\<.*\>)\s*""?(?<Message>.*)\""?", serverId),
-                });
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-add\s*warn\s*-user\s*(?<User><@\d+>)\s*-reason\s*(?<Reason>.+)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.WarnsCommand", @"-warns\s*(-user\s*(?<User><@&?\d+>))?\s*(?<All>(-all)?)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Messaging.BotCommands.SendCommand", @"-send\s*(?<Channel>\<.*\>)\s*""?(?<Message>.*)\""?", serverId)                });
```

Comment:
`<.*\>`
jak dobrze pamiętam to ma być `\<#&?\d+\>`
ale sprawdź tak dla pewności
chcemy mieć pewność że tam jest ID, a nie że ktoś wrzucił tekst między <>

Reply:
SendCommand to nie ja dodałem, widocznie wrzuciło mi się jak dawałem git pulla